app/components/bookings/create/create-booking.component.ts

```diff
@@ -1,9 +1,130 @@
 import {Component} from '@angular/core';
+import {Workpackage} from "../../../models/workpackage/workpackage";
+import {NavParams, NavController, Alert} from "ionic-angular/index";
+import {BookingService} from "../../../services/bookings/booking.service";
+import {AbstractControl, ControlGroup, FormBuilder, Validators} from "@angular/common";
+import moment = require("moment/moment");
+import {BookingSessionService} from "../../../services/bookings/booking-session.service";
+import {BookingSession, Booking} from "../../../models/booking/booking";
+import Moment = moment.Moment;
+import {Response} from "@angular/http";
 
 @Component({
   templateUrl: 'build/components/bookings/create/create-booking.component.html'
 })
 export class CreateBookingComponent {
-  constructor() {
+  private _workpackage: Workpackage;
+  private _bookingSession: BookingSession;
+  effort : AbstractControl;
+  description: AbstractControl;
+  date: AbstractControl;
+  newEtc: AbstractControl;
+  authForm: ControlGroup;
+  myDate = moment().startOf('day').format('YYYY-MM-DD');
+  myEffort = moment().startOf('day').format('HH:mm');
+  myEtc: number;
+
+  constructor(navParams: NavParams,
+              private _bookingsService: BookingService,
+              private _bookingSessionService: BookingSessionService,
+              private _fb: FormBuilder,
```

Comment:
I think names like `_fb` make code generally harder to read. If you try to understand a method without having the complete context, it's much easier if the property is called something like `_formBuilder` instead.